client/layout/index.jsx

```diff
@@ -198,6 +173,26 @@ function SidebarOverflowDelay( { layoutFocus } ) {
 	return null;
 }
 
+function AppBannerLoader( siteId ) {
```

Comment:
I createed this functional component so that I could access the logic from `useShouldShowCriticalAnnouncementsQuery`